.npmrc

```
@@ -1,5 +1,3 @@
-node-version=20.11.1
-engine-strict=true
 ignore-scripts=true
```

```suggestion
engine-strict=true
ignore-scripts=true
```

Refs: https://pnpm.io/9.x/npmrc#engine-strict

i just want to test something w/ this, but one of the biggest reasons why this field is kinda lame is bc project authors often incorrectly set this when the dep still works fine for us

---

i think we will just keep it since it's not interacting badly anymore